util/entitlements/security.go

```diff
@@ -0,0 +1,74 @@
+package entitlements
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/oci"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// WithUnconfinedSpec sets the default seccomp profile to the spec with Admin capability.
+// Note: must follow the setting of process capabilities
+func WithUnconfinedSpec() oci.SpecOpts {
```

Comment:
This code should be in a Linux specific file, as all the operations only apply to a Linux spec.

Reply:
I renamed the file to `secuirty_linux.go`